src/test/scala/mesosphere/marathon/LaunchQueueModuleTest.scala

```diff
@@ -199,7 +198,7 @@ class LaunchQueueModuleTest extends AkkaUnitTest with OfferMatcherSpec {
       val request = InstanceOpFactory.Request(app, offer, Map.empty, additionalLaunches = 1)
       verify(instanceOpFactory).matchOfferRequest(request)
       matchedTasks.offerId should equal(offer.getId)
-      launchedTaskInfos(matchedTasks) should equal(Seq(mesosTask))
+      matchedTasks.ops.collect{ case e: InstanceOp.LaunchTask => e }.map(_.taskInfo) should contain theSameElementsAs Seq(mesosTask)
```

Comment:
I like the `should contain theSameElementsAs` usage but `collect`-ing the ops in every test... I don't see how that's an improvement.

Reply:
Less code but yeah. It annoys me as well.